railties/lib/rails/generators/rails/encrypted_file/encrypted_file_generator.rb

```diff
@@ -1,38 +0,0 @@
-# frozen_string_literal: true
-
-require "rails/generators/base"
-require "active_support/encrypted_file"
-
-module Rails
-  module Generators
-    class EncryptedFileGenerator < Base
-      def add_encrypted_file(file_path, key_path)
-        unless File.exist?(file_path)
-          say "Adding #{file_path} to store encrypted content."
-          say ""
-          say "The following content has been encrypted with the encryption key:"
-          say ""
-          say template, :on_green
-          say ""
-
-          add_encrypted_file_silently(file_path, key_path)
-
-          say "You can edit encrypted file with `bin/rails encrypted:edit #{file_path}`."
-          say ""
-        end
-      end
-
-      def add_encrypted_file_silently(file_path, key_path, template = encrypted_file_template)
-        unless File.exist?(file_path)
-          setup = { content_path: file_path, key_path: key_path, env_key: "RAILS_MASTER_KEY" }
-          ActiveSupport::EncryptedFile.new(setup).write(template)
-        end
-      end
-
-      private
-        def encrypted_file_template
-          "# aws:\n#  access_key_id: 123\n#  secret_access_key: 345\n\n"
-        end
-    end
-  end
-end
```

Comment:
Why are we deleting this?

Reply:
It's no longer in use after replacing it with simple 4 lines here https://github.com/rails/rails/pull/31276/files#diff-879c12dde8f5250250844e7741912126R52;
And `add_encrypted_file` was never used before.

Reply:
In order to match the hierarchy of class with other commands, I would like to keep `EncryptedFileGenerator`. 
Can you change to remove only `add_encrypted_file` method?

Reply:
@y-yagi Done. Looks like failures on Travis are not related to this PR.